api-mock/fetcher.ts

```diff
@@ -0,0 +1,248 @@
+import { fetch } from "cross-fetch";
+import { MockedVars } from "./api-mock-server";
+import { PROCESS_STATE } from "supertokens-node/lib/build/processState";
+import { serializeVars } from "./utils";
+
+const apiMockPort = process.env.ST_SDK === "golang" ? 3032 : process.env.ST_SDK === "python" ? 3031 : 3030;
```

Comment:
I'd prefer if this was controlled by a single `API_PORT` var

Reply:
agreed, changed